xml/article_nfs_storage.xml

```diff
@@ -450,255 +465,240 @@ include /etc/drbd.d/*.res;</screen>
-       verify this:
-      </para>
-      <screen>&prompt.root;<command>exportfs</command> -v
-/srv/nfs/work   <replaceable>IP_ADDRESS_OF_CLIENT</replaceable>(...)
-/srv/nfs        <replaceable>IP_ADDRESS_OF_CLIENT</replaceable>(...)</screen>
-     </listitem>
-    </orderedlist>
-   </sect3>
--->
-
-<!--   <sect3 xml:id="sec-ha-quick-nfs-resources-nfsexport-nonroot">
-    <title>NFS exports</title>-->
-    <para>
-     To export the <filename>/srv/nfs/work</filename> directory to clients,
-     use the following primitive:
-    </para>
-    <orderedlist>
-     <listitem>
-      <para>
-       Create NFS exports with the following commands:
-      </para>
-<screen>&prompt.crm.conf;<command>primitive exportfs_work \
-  ocf:heartbeat:exportfs \
-    params directory="/srv/nfs/work" \
-      options="rw,mountpoint" \
-      clientspec="&nfs-clientspec;" \
-      wait_for_leasetime_on_stop=true \
-      fsid=100 \
-  op monitor interval="30s"</command></screen>
-     </listitem>
-     <listitem>
-      <para>
-       After you have created these resources, append them to the existing
-       <literal>g-nfs</literal> resource group:
-      </para>
-<screen>&prompt.crm.conf;<command>modgroup g-nfs add exportfs_work</command></screen>
-     </listitem>
-     <listitem>
-      <para>
-       Commit this configuration:
-      </para>
-<screen>&prompt.crm.conf;<command>commit</command></screen>
+   <procedure>
+    <title>Creating supporting resources for NFS</title>
+    <step>
+     <para>
+      Create a primitive to manage the NFS server daemon:
+     </para>
+<screen>&prompt.crm.conf;<command>primitive p_nfsserver nfsserver \
+  params nfs_server_scope=SUSE \
+  op monitor interval=10s timeout=20s</command></screen>
+    </step>
+     <step>
       <para>
-       Pacemaker will export the NFS virtual file system root and the two
-       other exports.
+       Create a primitive for the NFS exports:
       </para>
-     </listitem>
-     <listitem>
+<screen>&prompt.crm.conf;<command>primitive p_exportfs exportfs \
+  params directory="/srv/nfs/share" \
+  options="rw,mountpoint" clientspec="*" fsid=100 \
+  op monitor interval=30s</command></screen>
+      <important>
+       <title>Do not set <literal>wait_for_leasetime_on_stop=true</literal></title>
+       <para>
+        Setting this option to <literal>true</literal>
+        in a highly available NFS setup can cause unnecessary delays and loss of locks.
+       </para>
+       <para>
+        The default value for <literal>wait_for_leasetime_on_stop</literal> is
+        <literal>false</literal>. There is no need to set it to <literal>true</literal>
+        when <filename>/var/lib/nfs</filename> and <literal>p_nfsserver</literal>
+        are configured as described in this guide.
+       </para>
+      </important>
+     </step>
+     <step>
       <para>
        Confirm that the NFS exports are set up properly:
       </para>
 <screen>&prompt.root;<command>exportfs -v</command>
```

Comment:
Move this step to verify the NFS export later, only after `sudo crm configure commit`.

Reply:
I wondered about that also, this time through.   Good catch.